webrender/src/render_backend.rs

```diff
@@ -401,8 +401,7 @@ impl RenderBackend {
                         notifier.as_mut()
                                 .unwrap()
                                 .as_mut()
-                                .unwrap()
-                                .pipeline_size_changed(pipeline_id, Some(new_size));
+                                .unwrap();
```

Comment:
Sorry I'm just struggling to get this, it's the first time I've ever seen this code :-/, Is it basically everything to do with `updated_pipeline_sizes` that can be removed? So pretty much everything bellow `self.frame.create(&self.scene, &mut new_pipeline_sizes);`